### DISCUSSION TITLE
Is there any way add shapes to images

### DISCUSSION THREAD

**Discussion by Aruljebaraj** [hm21/pro_image_editor#114] [Q&A] [answered]

**Comment by hm21** [comment, marked as answer]
I moved your questions to the Discuss tab because that tab is better for Q&A.

What do you mean by adding shapes to the image? 

Do you mean, for example, your image will look like a star or circle?  If yes, then yes it's possible you can add “stickers” which can be any widget you want. You can check out this [example](https://github.com/hm21/pro_image_editor/blob/stable/example/lib/features/stickers_example.dart), which shows you how to add stickers. For the case you want to add shapes to the background image, you can follow this [example](https://github.com/hm21/pro_image_editor/blob/stable/example/lib/features/movable_background_image.dart) because you need to convert it to a sticker first.

Or maybe you mean drawing things like arrows, which is possible in the Painting Editor.

**Reply by Aruljebaraj** [reply]
would be better if user can add geometric shapes eg:  circle ,arrow, square,line

**Comment by hm21** [comment]
Closed due to no response from author.
